src/generators/modelsGenerator.ts

```diff
@@ -69,6 +69,10 @@ export function generateModels(clientDetails: ClientDetails, project: Project) {
       namespaceImport: "coreRestPipeline",
       moduleSpecifier: "@azure/core-rest-pipeline"
     });
+    modelsIndexFile.addImportDeclaration({

```

Comment:
Do we need to check for `compatMode` before importing this?

Reply:
Actually nope. All unused imports will be removed at the end by the prettier.